.github/workflows/job_pytorch_layer_tests.yml

```diff
@@ -0,0 +1,167 @@
+name: PyTorch Layer Tests
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: 'Machine on which the tests would run'
+        type: string
+        required: true
+      shell:
+        description: "shell to override the default shell settings in the runner's operating system."
+        type: string
+        required: true
+      container:
+        description: 'JSON to be converted to the value of the "container" configuration for the job'
+        type: string
+        required: false
+        default: '{"image": null}'
+      affected-components:
+        description: 'Components that are affected by changes in the commit defined by the Smart CI Action'
+        type: string
+        required: true
+
+permissions: read-all
+
+env:
+  PIP_CACHE_PATH: /mount/caches/pip/linux
+  PYTHON_VERSION: '3.11'
```

Comment:
I suggest moving it to the `inputs` like in https://github.com/openvinotoolkit/openvino/blob/2507d89d9cfcf38bf43238aaf83ff74f066fe452/.github/workflows/job_python_unit_tests.yml#L19 so that it would be possible to test on different Python versions in different workflows.

Reply:
let me fix in separate PR.